components/home/LatestNews.vue

```diff
@@ -35,44 +35,10 @@
 </template>
 
 <script setup lang="ts">
-import gql from "graphql-tag";
+import { getPosts } from "@/components/blog";
 
-// The subsocial space for news: https://polkaverse.com/10802 , and Japanese: https://polkaverse.com/11315
-const { locale } = useI18n();
-const astarSpace = locale.value === "ja" ? 11315 : 10802;
-const query = gql`
-  query PostsBySpaceId {
-    posts(where: { space: { id_eq: "${astarSpace}" }, hidden_eq: false }, orderBy: id_DESC, limit: 12) {
-      publishedDate: createdOnDay
-      title
-      href: canonical
-      image
-      slug
-      id
-    }
-  }
-`;
-
-const { data } = await useAsyncQuery({ query, clientId: "subsocial" });
-const posts = data.value.posts.map(
-  (item: { publishedDate: string | number | Date }) => {
-    const lowercaseSlug = item.slug.toLowerCase();
-    const date = new Date(item.publishedDate);
-    const formattedDate = date.toLocaleDateString("en-US", {
-      month: "short",
-      day: "numeric",
-      year: "numeric",
-    });
-    return {
-      ...item,
-      image: item.image
-        ? "https://ipfs.subsocial.network/ipfs/" + item.image
-        : "/images/blog/placeholder.webp",
-      publishedDate: formattedDate,
-      slug: lowercaseSlug,
-    };
-  }
-);
+const pagination = "limit: 4";
```

Comment:
```suggestion
const pagination = "limit: 12";
```
Please set this limit to 12. The blog posts are displayed as a carousel.
<img width="1624" alt="Screenshot 2024-03-21 at 9 28 44 AM" src="https://github.com/AstarNetwork/astarwebsite_v2/assets/33358068/36c020b1-3448-4c34-b5c4-831080280479">